src/main/java/uz/pdp/eufloria/domain/Store.java

```diff
@@ -0,0 +1,38 @@
+package uz.pdp.eufloria.domain;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.time.LocalTime;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
```

Comment:
Why you need @ToString annotation ?

Reply:
I thought that after some time maybe it will be needful. But now I think it doesn't need

Reply:
ToString is removed